main.css

```diff
@@ -0,0 +1,115 @@
+/*
+gold color - #FBD784
+background - #0B1D26
+ */
+
+@font-face {
+    font-family: "LibreBaskerville-Bold";
+    src: url("./fonts/LibreBaskerville-Bold.ttf") format('ttf');
+}
+
+@font-face {
+    font-family: "LibreBaskerville-Italic";
+    src: url("./fonts/LibreBaskerville-Italic.ttf") format('ttf');
+}
+
+@font-face {
+    font-family: "LibreBaskerville-Regular";
+    src: url("./fonts/LibreBaskerville-Regular.ttf") format('ttf');
+}
+
+@font-face {
+    font-family: "Chronicle-Display-Bold";
+    src: url("./fonts/Chronicle-Display-Bold.otf") format('otf');
+}
+
+@font-face {
+    font-family: "Chronicle-Display-Semibold";
+    src: url("./fonts/Chronicle-Display-Semibold.otf") format('otf');
+}
+
+* {
+    box-sizing: border-box;
+    margin: 0px;
+    padding: 0px;
+}
+
+main {
+    background-color: #0B1D26;
+    color: white;
+    display: flex;
+    flex-direction: column;
+    align-items: column;
+    font-size: 62.5%;
+}
+.main__list {
+    max-width: 1462px;
+    margin: 0 auto;
+    padding: 0 2rem;
+    margin-bottom: 20REM;
+}
+
+.main:last-child {
+    margin: 0;
+}
+
+.item {
+    display: flex;
+    flex-direction: space-between;
+    gap: 4rem;
+    max-width: 1462px;
+}
+
+.item__info {
+    position: relative;
+    margin: auto;
+    padding-left: 15rem;
+    max-width: 782px;
+}
+
+.item__count {
+    font-size: 24rem;
+    font-weight: 700;
+    opacity: 10%;
+    position: absolute;
+    left: 0;
+    top: -103px;
+}
+
+.item__subtitle {
+    margin-bottom: 27px;
+}
+
+
+.item__title {
+    font-size: 5rem;
+}
+
+.item__text {
+    font-size: 1.8rem;
+    padding-top: 2.2rem;
+}
+
+.item__link {
+    font-size: 2rem;
+    background-color: #0000;
+    color: #FBD784 ;
+    text-align: left;
+    padding-top: 2.7rem;
+}
+
+.item__fig {
+    max-width: 566px;
+
+}
+
+.img__fig {
+    max-width: none;
+}
+
+
+button {
```

Comment:
To się tyczy wszystkich buttonów na stronie? Ponieważ używasz selektora ogólnego typu